--- linux-2.6.35.12-cavm1/drivers/net/cns3xxx/cns3xxx_ethtool.c	
+++ linux-2.6.35.12-cavm1/drivers/net/cns3xxx/cns3xxx_ethtool.c
@@ -26,8 +26,8 @@
 #include <linux/types.h>
 #include <linux/ethtool.h>
 #include <linux/netdevice.h>
-#include "../cns3xxx_pse_init/cns3xxx_symbol.h"
-#include "../cns3xxx_pse_init/cns3xxx.h"
+#include "../../cns3xxx_pse_init/cns3xxx_symbol.h"
+#include "../../cns3xxx_pse_init/cns3xxx.h"
 #include "cns3xxx_tool.h"
 
 #ifdef CONFIG_CNS3XXX_PSE_SW_PHY_AUTO_POLLING
